ios/browser/api/bookmarks/brave_bookmarks_api.h

```diff
@@ -0,0 +1,113 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_IOS_BROWSER_API_BOOKMARKS_BRAVE_BOOKMARKS_API_H_
+#define BRAVE_IOS_BROWSER_API_BOOKMARKS_BRAVE_BOOKMARKS_API_H_
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+typedef NS_ENUM(NSUInteger, BookmarksNodeType) {
+  URL,
+  FOLDER,
+  BOOKMARK_BAR,
+  OTHER_NODE,
+  MOBILE
```

Comment:
Enum's in Obj-C aren't namespaced, shouldn't these be prefixed by `BookmarksNodeType`, like `BookmarksNodeTypeURL`, `BookmarksNodeTypeFolder`, etc.?